priv/static/specs/http_api_spec.yaml

```diff
@@ -55,6 +55,25 @@ paths:
   /contexts:
     delete:
       summary: 'delete all contexts'
+      parameters:
+        - name: version
+          in: query
+          required: false
+          type: string
+          description: 'Version'
+          default: 'v1'
+        - name: imsi
+          in: query
+          required: false
+          type: integer
+          description: 'International Mobile Subscriber Identity'
+          default: '111111111111111'
+        - name: pfcp
+          in: query
+          required: false
+          type: string
+          description: 'PFCP'
+          default: 'topon.sx.prox01.epc.mnc001.mcc001.3gppnetwork.org'
```

Comment:
The example might be misleading. The FQDN represents the name of a specific interface on an UPF instance, the FQDN of the UPF instance itself would be different. In this case the UPF instance name is most likely `prox01.epc.mnc001.mcc001.3gppnetwork.org`